krun.py

```diff
@@ -232,7 +229,7 @@ def run(self):
 
         info("Done: Results dumped to %s" % self.out_file)
         if errors:
-            warn("%s ERRORS OCCURRED! READ THE LOG!%s" % (ANSI_RED, ANSI_RESET))
+            warn("Errors occurred! Read the scrollback!")
```

Comment:
What happens if errors exceed the scrollback? I thought the point was that we dumped these things to a log file and/or emailed them to a given address too?